src/RadioGroup.js

```diff
@@ -12,7 +12,7 @@ class RadioGroup extends React.Component {
         }),
         container: PropTypes.string,
         name: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
```

Comment:
any reason why it's not required anymore?
Usually, React wants us to provide an `onChange` callback if we set a `value`.
This is valable for every input.